### PR TITLE
testcase/kernel/tc_libc_string : Refactoring tc_libc_string

### PR DESCRIPTION
**Pull request by hasw7569**
Change strlen("string") + 1 to sizeof("string") and change "Bad address" to EFAULT_STR

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [0a6232155b25f5a67d505e8f7b2714a628448053] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [0a6232155b25f5a67d505e8f7b2714a628448053] - Code Rule Check OK.

File: apps/examples/testcase/le_tc/kernel/tc_libc_string.c [hunk 1]
```diff
@@ -39,9 +39,9 @@
 
 #define BUFF_SIZE 5
 #define BUFF_SIZE_10 10
-#define BUFF_SIZE_12 12
 
 #define EBUSY_STR_SIZE (sizeof(EBUSY_STR))
+#define EFAULT_STR_SIZE (sizeof(EFAULT_STR))
```
Comment (jeongchanKim) [inline apps/examples/testcase/le_tc/kernel/tc_libc_string.c]: EFAULT_STR_SIZE is used just once. How about using directly, not macro?

Comment (hasw7569) [reply apps/examples/testcase/le_tc/kernel/tc_libc_string.c]: I agree.
I will change sizeof(EFAULT_STR) instead of macro

**Comment by seinfra** [comment]
Target : [519fc73654e8973bb2668b38f977b9f532f88ba3] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [519fc73654e8973bb2668b38f977b9f532f88ba3] - Code Rule Check OK.

**Pull request merged**